tests/test_tts.py

```diff
@@ -43,7 +43,7 @@ async def test_async_setup_entry(
         result = await async_setup_entry(hass, config_entry)
 
         # Verify that the function returns the expected value
-        assert result == True
+        assert result is True
```

Comment:
could you take a stab at getting tests to pass again? it fails currently due to changes in this pr (get_engine is gone).